regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,183 @@
+"""Helper for finding and listing contacts.
+
+   Projecta are small bite-sized project quanta that typically will result in
+   one manuscript.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from pathlib import Path
+from regolith.dates import get_due_date
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+THIS_DIR = Path('.')
+inst_file = THIS_DIR / ".." / ".." / ".." / ".." / "dbs" / "rg-db-public" / "db" / "institutions.yml"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='match contacts to a name fragment')
+    subpi.add_argument("-i", "--inst", help='match people to institutions')
+    subpi.add_argument("-d", "--day", help='match people to days')
```

Comment:
The date logic is not right.  accept ``-d --date`` and ask for ISO format.  Then we can discuss how large of a range we want to create around that date.  Maybe make a default range (perhaps +/- 2 months around date?) but have an optional "-r --range" so the user can override the range.  Ask for a range in months and in the help message specify if it is +/- r or, probably better, a range of `r`` centered on d, so it would be ``d +/- r/2``